contracts/src/v0.8/l2ep/dev/scroll/ScrollSequencerUptimeFeed.sol

```diff
@@ -130,10 +125,9 @@ contract ScrollSequencerUptimeFeed is
   /// @param roundId The round ID to update
   /// @param status Sequencer status
   function _updateRound(uint80 roundId, bool status) private {
-    uint64 updatedAt = uint64(block.timestamp);
-    s_rounds[roundId].updatedAt = updatedAt;
-    s_feedState.updatedAt = updatedAt;
-    emit RoundUpdated(_getStatusAnswer(status), updatedAt);
+    s_feedState.updatedAt = uint64(block.timestamp);
+    s_rounds[roundId].updatedAt = s_feedState.updatedAt;
```

Comment:
You're referencing a storage var here, over `uint64(block.timestamp)`, which would be a lot cheaper. Not sure if the compiler fixes it for you but let's change it because it looks wrong. Same below.